quickwit/quickwit-codegen/example/Cargo.toml

```diff
@@ -30,12 +31,10 @@ quickwit-common = { workspace = true }
 quickwit-proto = { workspace = true }
 
 [dev-dependencies]
-mockall = { workspace = true }
-
```

Comment:
removing this breaks running `cargo nextest run` from `:/quickwit/quickwit-codegen/example`